streaming/src/main/scala/org/apache/spark/streaming/dstream/ReceiverInputDStream.scala

```diff
@@ -41,6 +42,12 @@ abstract class ReceiverInputDStream[T: ClassTag](@transient ssc_ : StreamingCont
   extends InputDStream[T](ssc_) {
 
   /**
+   * Asynchronously maintains & sends new rate limits to the receiver through the receiver tracker.
+   */
+  override protected[streaming] val rateController: Option[RateController] =
```

Comment:
This should change a little. See comment on what the config params should be in another comment below (near `RateEstimator.makeEstimator`)